content/articles/jach/jach_006.rst

```diff
@@ -0,0 +1,20 @@
+Fuzzing: Hack, Art, and Science
+###############################
+
+:date: 2020-1-27
+:category: Paper Review
+:author: Joseph Anthony C. Hermocilla
+
+A program usually accepts input, performs operations on the input, and produces output.  Incorrect processing of input can lead to security vulnerabilities, such as buffer overflow. Depending on the input, the program may not perform the desired operation. An attacker can craft a specialized input that exploits the vulnerability to take control of the machine running the vulnerable software. 
+
+This paper [GODEFROID2020] gives an overview of fuzzing as a way to detect security vulnerabilities. The other two approaches mentioned in the paper include using static program analyzers and doing manual code inspection. Fuzzing tests the behavior of a program against all, or a subset, of possible inputs to identify security vulnerabilities. It is an automated testing technique since there is a large set of possible inputs to a program. 
```

Comment:
Other approaches mentioned in the paper include the use of static program analyzers and manual code inspection.